mobile/library/common/jni/BUILD

```diff
@@ -53,10 +53,7 @@ cc_library(
     srcs = [
         "jni_interface.cc",
     ],
-    copts = envoy_select_envoy_mobile_request_compression(
-        ["-DENVOY_MOBILE_REQUEST_COMPRESSION"],
-        "@envoy",
-    ),
+    copts = envoy_mobile_copts("@envoy"),
```

Comment:
can we just move this over to envoy_cc_library?  If not maybe comment why?

Reply:
I'll try moving it over. I _think_ it should be safe.